### PR TITLE
SAAS-374 logout with Cypress Actions

### PR DESCRIPTION
**Pull request by yurkovychv**

### CODE REVIEW

File: packages/platform-ui/cypress/integration/auth/login.spec.ts [hunk 1]
```diff
@@ -1,29 +1,40 @@
...
+import { popUp } from 'pages/common/view/selectors';
 
 context('Login', () => {
   beforeEach(() => {
+    setAliases();
     cy.visit(pageDetailsMap[Pages.Login].url);
   });
 
-  it('should be able to see the login form', () => {
+  it('SAAS-T82 - should be able to see the login form', () => {
```
Comment (nicolalamacchia) [inline packages/platform-ui/cypress/integration/auth/login.spec.ts:17]: What are these SAAS codes?

**Review by nicolalamacchia — APPROVED** [review]

File: packages/platform-ui/cypress/integration/auth/login.spec.ts (hunk 1 above)
Comment (yurkovychv) [reply packages/platform-ui/cypress/integration/auth/login.spec.ts:17]: These are test case IDs. It is easier to find a realted test case or find a test by test case ID.

File: packages/platform-ui/cypress/pages/auth/actions/signup.action.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
+/// <reference types="cypress" />
+import { User } from 'pages/common/interfaces/Auth';
+import { authSignupAction } from '../../../../src/store/auth';
```
Comment (atymchuk) [inline packages/platform-ui/cypress/pages/auth/actions/signup.action.ts:3]:
```suggestion
import { authSignupAction } from 'store/auth';
```
nit: it should be possible ^

File: packages/platform-ui/src/index.tsx [hunk 3]
```diff
@@ -57,3 +57,12 @@ ReactDOM.render(
 // unregister() to register() below. Note this comes with some pitfalls.
 // Learn more about service workers: https://bit.ly/CRA-PWA
 serviceWorker.unregister();
+
+// expose store during tests
```
Comment (atymchuk) [inline packages/platform-ui/src/index.tsx:61]: Could you please put a todo for FE?
// TODO: add `Cypress` and `store` to the `window` type.

**Review by atymchuk — DISMISSED** [review]

File: packages/platform-ui/cypress/pages/auth/actions/signup.action.ts (hunk 2 above)
Comment (yurkovychv) [reply packages/platform-ui/cypress/pages/auth/actions/signup.action.ts]: it did not work, I tried. I've added `paths` to `tsconfig.json` file and changed import to `@src/store/auth`

File: packages/platform-ui/src/index.tsx (hunk 3 above)
Comment (yurkovychv) [reply packages/platform-ui/src/index.tsx:61]: done, thanks

**Review by atymchuk — APPROVED** [review]

**Pull request merged**